OrbitCore/FramePointerValidator.cpp

```diff
@@ -16,11 +17,13 @@ FramePointerValidator::GetFpoFunctions(
   std::vector<std::shared_ptr<Function>> result;
 
   cs_mode mode = is_64_bit ? CS_MODE_64 : CS_MODE_32;
-  csh handle;
-  if (cs_open(CS_ARCH_X86, mode, &handle) != CS_ERR_OK) {
+  csh temp_handle;
+  if (cs_open(CS_ARCH_X86, mode, &temp_handle) != CS_ERR_OK) {
     ERROR("Unable to open capstone.");
     return {};
```

Comment:
shouldn't we also close here?

Reply:
No, in case of an error the handle (and the underlying data structure) was never initialized, so we are fine here.